projects/1.18-fabric/assets/sodium/sodium/lang/zh_cn.json

```
@@ -1,66 +1,66 @@
-  "sodium.options.vignette.tooltip": "如果启用，将在玩家的画面上显示晕影效果。除非你的显卡像素填充率有限，否则这不太可能对帧率产生影响。",
-  "sodium.options.mipmap_levels.tooltip": "控制将用于方块模型纹理的MipMap的数量。较高的值可以更好地渲染远处的方块，但可能会对许多纹理动画的性能产生不利影响。",
-  "sodium.options.use_block_face_culling.name": "使用方块面剔除",
-  "sodium.options.use_block_face_culling.tooltip": "如果启用，则只会提交面向相机一侧的方块的面进行渲染。这可以在渲染过程的早期消除大量方块面，从而节省GPU上的内存带宽和时间。某些资源包可能会在此选项上出现问题，因此如果你在方块中看到空洞，请尝试禁用它。",
-  "sodium.options.use_fog_occlusion.name": "使用雾遮挡剔除",
-  "sodium.options.use_fog_occlusion.tooltip": "如果启用，确定被雾效果完全遮挡的块将不会被渲染，有助于提高性能。当雾效果较重时（例如在水下时），改进可能会更加显著，但在某些情况下可能会导致天空和雾之间出现不良的视觉瑕疵。",
-  "sodium.options.use_entity_culling.name": "使用实体剔除",
-  "sodium.options.use_entity_culling.tooltip": "如果启用，则在渲染期间将跳过确定不在任何可见区块中的实体。这可以通过避免渲染位于地下或墙后的实体来帮助提高性能。",
-  "sodium.options.use_particle_culling.name": "使用粒子剔除",
-  "sodium.options.use_particle_culling.tooltip": "如果启用，则只会渲染确定为可见的粒子。当附近有许多粒子时，这可以显着提高帧率。",
-  "sodium.options.animate_only_visible_textures.name": "仅可见纹理有动画",
-  "sodium.options.animate_only_visible_textures.tooltip": "如果启用，则只会更新确定为可见的纹理动画。这可以显着提高某些硬件的帧率（FPS），尤其是对于大型的资源包。如果你遇到某些纹理失去动画的问题，请尝试禁用此选项。",
-  "sodium.options.cpu_render_ahead_limit.name": "CPU预渲染限制",
-  "sodium.options.cpu_render_ahead_limit.tooltip": "指定CPU可以在GPU上等待完成渲染的最大帧数。非常低或非常高的值可能会导致帧率（FPS）不稳定。",
-  "sodium.options.cpu_render_ahead_limit.value": "%s帧",
-  "sodium.options.allow_direct_memory_access.name": "允许直接内存访问",
-  "sodium.options.allow_direct_memory_access.tooltip": "如果启用，将允许某些关键代码路径使用直接内存访问来提高性能。这通常会大大减少区块和实体渲染的CPU开销，但会使诊断某些错误和崩溃变得更加困难。只有你被要求这么做或知道你在做什么，才应该禁用它。",
-  "sodium.options.enable_memory_tracing.name": "启用内存跟踪",
-  "sodium.options.enable_memory_tracing.tooltip": "调试功能。如果启用，堆栈跟踪将与内存分配一起收集，以帮助在检测到内存泄漏时改进诊断信息。",
-  "sodium.options.performance_impact_string": "性能影响：%s",
-  "sodium.options.use_persistent_mapping.name": "使用持久映射",
-  "sodium.options.use_persistent_mapping.tooltip": "如果启用，少量内存将被持久映射为区块上传的暂存缓冲区，有助于减少加载或更新区块时的CPU开销和帧时间不稳定。\n需要OpenGL 4.4或ARB_buffer_storage拓展。",
-  "sodium.options.chunk_memory_allocator.name": "区块内存分配器",
-  "sodium.options.chunk_memory_allocator.tooltip": "选择将用于区块渲染的内存分配器。\n1、Async：最快，适用于大多数现代图形驱动程序。\n2、Swap：旧图形驱动程序的兼容选项。可能会显着增加内存使用量。",
-  "sodium.options.chunk_memory_allocator.async": "Async",
-  "sodium.options.chunk_memory_allocator.swap": "Swap",
-  "sodium.options.chunk_update_threads.name": "区块更新线程",
-  "sodium.options.chunk_update_threads.tooltip": "指定用于区块构建的线程数。使用更多线程可以加快区块加载和更新的速度，但可能会对帧时间产生负面影响。",
-  "sodium.options.always_defer_chunk_updates.name": "始终推迟区块更新",
-  "sodium.options.always_defer_chunk_updates.tooltip": "如果启用，渲染将永远不会等待区块更新完成，即使它们很重要。这在某些情况下可以极大地提高帧率（FPS），但它可能会在世界上产生明显的视觉滞后。",
-  "sodium.options.buttons.undo": "撤销",
-  "sodium.options.buttons.apply": "应用",
-  "sodium.options.buttons.donate": "给我们买杯咖啡！"
+{
+  "sodium.option_impact.low": "低",
+  "sodium.option_impact.medium": "中",
+  "sodium.option_impact.high": "高",
+  "sodium.option_impact.extreme": "极高",
+  "sodium.option_impact.varies": "视情况而定",
```

Varies 是 `to change or be different according to the situation`，视情况而定太长了，建议意译为自动，和其他位置也统一
```suggestion
  "sodium.option_impact.varies": "自动",
```